src/telemetry/cookies.ts

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
This file duplicates https://sourcegraph.com/github.com/sourcegraph/sourcegraph@90826c6d7370511469e528258ee6ee446f784870/-/blob/client/shared/src/telemetry/web/cookies.ts from the sourcegraph/sourcegraph repo